### ISSUE TITLE
Customized unit

### ISSUE THREAD

**Issue by lkuiucsb** [BLE-LTER/MetaEgress#46]
The current code doesn't work for the customized unit. For example, if you put it an unit "milligramPerGramPerHourPerPhotonFlux", the create_entity function fails. 

I run the code on Mac.

**Comment by lkuiucsb** [comment]
I just add one thing here. The same code works on Windows. So there is something about the "set_attributes()" function from the EML 2 package doesn't work the same between Mac and PC.

**Comment by atn38** [comment]
Thank you Li for the report. Can you report the error message on Macs? If
you suspect it's because of set_attributes, then we should report to the
EML R folks.
This is potentially a bigger issue as MetaEgress isn't tested on really
anything but my version of Windows yet.

**Comment by lkuiucsb** [comment]
yes, probably a good idea to report to EML R team. I could do that.

**Comment by lkuiucsb** [comment]
There was actually an issue couple months ago about this: https://github.com/ropensci/EML/issues/278

**Comment by atn38** [comment]
Thanks for taking care of it. I haven't run into this issue myself, but it
seems people do run into it on Windows, not just Mac.
